test/NonSiloTests/General/Identifiertests.cs

```diff
@@ -12,19 +12,20 @@
 
 namespace UnitTests.General
 {
+    [Collection(TestEnvironmentFixture.DefaultCollection)]
     public class Identifiertests
     {
         private readonly ITestOutputHelper output;
+        private readonly TestEnvironmentFixture environment;
         private static readonly Random random = new Random();
 
         class A { }
         class B : A { }
         
-        public Identifiertests(ITestOutputHelper output)
+        public Identifiertests(ITestOutputHelper output, TestEnvironmentFixture fixture)
```

Comment:
same, no need to add the fixture to the constructor if you do not intend to access it from the tests